src/analysis/locally_convex/basic.lean

```diff
@@ -132,9 +132,15 @@ def balanced (A : set E) := ∀ a : 𝕜, ∥a∥ ≤ 1 → a • A ⊆ A
 
 variables {𝕜}
 
-lemma balanced_mem {s : set E} (hs : balanced 𝕜 s) {x : E} (hx : x ∈ s) {a : 𝕜} (ha : ∥a∥ ≤ 1) :
+@[simp] lemma balanced_empty : balanced 𝕜 (∅ : set E) :=
+λ _ _, by { rw smul_set_empty }
+
+lemma balanced_iff_smul_mem : balanced 𝕜 s ↔ ∀ ⦃a : 𝕜⦄, ∥a∥ ≤ 1 → ∀ ⦃x : E⦄, x ∈ s → a • x ∈ s :=
+forall₂_congr $ λ a ha, smul_set_subset_iff
+
+lemma balanced.smul_mem (hs : balanced 𝕜 s) {a : 𝕜} (ha : ∥a∥ ≤ 1) {x : E} (hx : x ∈ s) :
   a • x ∈ s :=
-mem_of_subset_of_mem (hs a ha) (smul_mem_smul_set hx)
+balanced_iff_smul_mem.mp hs ha hx
```

Comment:
This is what I mean by alias (sorry, Github doesn't let me select the two lines before).
```suggestion
alias balanced.smul_mem ↔ balanced.smul_mem _
```

Reply:
I have no idea how aliases work (and I could not find anything about them in TPIL).